app/controllers/rents_controller.rb

```diff
@@ -1,23 +1,26 @@
 class RentsController < ApiController
   include Wor::Paginate
   def create
-    rent = Rent.new(rent_params)
+    rent = Rent.new(rent_params.merge(user_id: params[:user_id]))
+    authorize rent
+
     if rent.save
-      render json: rent, status: :created
       RentWorker.perform_async(rent.id)
+      render json: rent, status: :created
     else
       render json: { error: rent.errors.messages }, status: :unprocessable_entity
     end
   end
 
   def index
+    authorize User.find(params[:user_id]), policy_class: RentPolicy
```

Comment:
What if you pass an `user_id` that does not exist? We need to handle that case. 

Hint: We've seen something similar before :)

Reply:
Also, there's no need to specify the policy_class since we are already in the RentsController.